api/v1beta1/metallb_types.go

```diff
@@ -45,7 +45,7 @@ type MetalLBSpec struct {
 	MetalLBImage string `json:"image,omitempty"`
 	// +optional
 	SpeakerNodeSelector map[string]string `json:"nodeSelector,omitempty"`
-	// tolerations is a list of tolerations applied to metalLB controller
+	// tolerations is a list of tolerations applied to MetalLB controller
```

Comment:
nit: change the doc to say this is applied to the speaker. Controller in this case is misleading

Reply:
Also, the spekaer is a daemonset